source/content/guides/new-relic/03-monitor-new-relic.md

```diff
@@ -45,11 +45,19 @@ For more information on using New Relic&reg;'s features, we encourage you to rev
 
 - [Troubleshooting WordPress Performance with New Relic&reg; Performance Monitoring](https://pantheon.io/blog/troubleshooting-wordpress-performance-new-relic)
 
-## Configure Ping Monitors for Availability
+## Configure Ping Monitors (Synthetics) for Availability
 
-New Relic provides a free availability monitoring service within their Synthetics tool suite at the Lite service level. This basic monitoring feature sends a request to designated URLs from configured locations. Requests are sent at configured intervals and alerts are sent via email when a response fails.
+New Relic provides an availability monitoring service within their Synthetics tool suite which can be enabled by [contacting support](/guides/support/contact-support/). This basic monitoring feature sends a request to designated URLs from configured locations. Requests are sent at configured intervals and alerts are sent via email when a response fails.
 
-Pantheon can provide New Relic Ping monitoring for free as part of the service. However, more advanced monitoring, including full browser testing and scripted interactions, is only available to customers on an annual contract and requires an additional cost. Contact our [sales team](https://pantheon.io/contact-us) or your dedicated account manager for more information.
+Pantheon can provide New Relic Synthetics ping monitoring as part of the service with a usage quota limited to 10K Synthetics checks per month for each Pantheon workspace.
+
+<Alert title="Note" type="info">
+
+When a Pantheon workspace has more than 10K Synthetics checks are running monthly Pantheon will adjust the frequency or number of locations a Synthetics check originates from the reduce the number of checks. This is required to allow Synthetic to continue to operate correctly under the overall New Relic quota for Pantheon.
```

Comment:
@tyherbert will our customers know that/when this reduction happens?

Reply:
Yes, we plan to notify the customer via email if a change has been made to their Synthetics monitors.

Reply:
So is the flow something like
* Pantheon staff somehow sees that a workspace has exceed 10K
* Pantheon staff manually adjust settings in New Relic
* Pantheon staff manually emails customer?

Who from the workspace will get that email?

Reply:
Correct. This is something that Manoj and I will be working on. We are following the customer communication flow (https://docs.google.com/document/d/1Myagjy-2JhfnG7AqLs6clQ2YV0jxRvz_6nxQK898RQg/edit#heading=h.2licx9cbwmja). Currently we send an email to the site owner and cc the admins (this is a process that Shaq and Manoj put together).